.env.test

```diff
@@ -21,4 +21,4 @@ RABBITMQ_VHOST=app
 REDIS_HOST=redis
 REDIS_PASSWORD=
 REDIS_PORT=6379
-REDIS_CACHE_DB=1
\ No newline at end of file
+REDIS_CACHE_DB=1
```

Comment:
no enter needed

Reply:
are u sure? github should show builtin warning